dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -204,7 +204,7 @@ const SavedRunsComponent = () => {
                           updateTblValue(val, NAME_KEY, item.resource_id)
                         }
                         toggleEdit={toggleEdit}
-                        onDateSelect={(_event, str) =>
+                        onDateSelect={(str) =>
                           updateTblValue(
```

Comment:
The call to `updateTblValue()` is going to result in a call to `dispatch()`...will that result in the page being re-rendered, which will result in the input focus being reset?  (If so, that would explain what we're seeing...although, it doesn't suggest a _solution_ for the problem....)

Reply:
The input focus reset is the problem with editing the dataset name. I'll try to fix it in the different PR.